### ISSUE TITLE
failed to extract archive downloaded

### ISSUE THREAD

**Issue by musm** [JuliaLang/julia#33591]
On 1.3 RC4 I'm seeing

```julia
 Resolving package versions...
┌ Warning: failed to extract archive downloaded from https://api.github.com/repos/JuliaArrays/StaticArrays.jl/tarball/1085ffbf5fd48fdba64ef8e902ca429c4e1212d3
└ @ Pkg.Operations C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\Operations.jl:494
┌ Warning: failed to extract archive downloaded from https://api.github.com/repos/climate-machine/Oceananigans.jl/tarball/1a6d2bb7e823e3a7169b4ff38c7181b95ec65fd6
└ @ Pkg.Operations C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\Operations.jl:494
┌ Warning: failed to extract archive downloaded from https://api.github.com/repos/JuliaGraphics/Luxor.jl/tarball/f46158af3a808740fb898f44d18728b8905c805d
└ @ Pkg.Operations C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\Operations.jl:494
┌ Warning: failed to extract archive downloaded from https://api.github.com/repos/JuliaStats/RDatasets.jl/tarball/a64599c0947dcc78edfa4f32559516b198ded29f
└ @ Pkg.Operations C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\Operations.jl:494
```

**Comment by staticfloat** [comment]
https://github.com/JuliaLang/julia/pull/33592

**Comment by Nosferican** [comment]
Was this fixed by #33592?

**Comment by staticfloat** [comment]
Yes, it should be.
